blockchain-scrapers/docker-compose.yml

```diff
@@ -96,6 +96,24 @@ services:
       restart_policy:
         delay: 2s
         window: 20s
+  tron:
+    image:
+      ${DOCKER_HUB_LOGIN}/tron
```

Comment:
you need to remove the ${DOCKER_HUB_LOGIN}, it gets me a "No such image", I don't have this image in my repository. what is the image you want to get ?

Reply:
Hey @frranck, 

You can target "patches12345/tron", the image is located here: https://hub.docker.com/r/patches12345/tron/

It's the image built from this repo: https://github.com/PaddyMc/go-tron/blob/master/Dockerfile

I can add my Docker Hub username to the docker-compose if you would like, but my advice would be to build the docker image linked above and upload it to your Docker Hub and use that for production. 

Let me know how you would like to proceed.

Reply:
Could you please include your docker file for tron into the project ?
then use:
image: ${DOCKER_HUB_LOGIN}/blockchain-scrapers_tron

Reply:
Ok, it's done. I've put the Tron node Dockerfile in the cmd/trx folder. And updated the tron image location to ${DOCKER_HUB_LOGIN}/blockchain-scrapers_tron

Just to be clear, there are two Dockerfiles now.

- one for the tron node (tron) => cmd/trx/Dockerfile used in the docker-compose.yml (line 101)
- one for the blockchain scraper (trx) => Dockerfile-trx used in the docker.compose.yml (line 218)

So in the the docker-compose there are two services, tron & trx

Let me know if there's anything else I can do to help, more than happy to.